### DISCUSSION TITLE
What happened to Wiz

### DISCUSSION THREAD

**Discussion by domingo316** [isaacrlevin/presencelight#824] [Q&A]
I asked for help setting up Wiz, I saw that it was supported but when I tried to do it, it didn't work and there is no documentation on how to set it up. My request was closed and said that if I have issues to open a bug. Now I go to try again to see if maybe I can figure it out and Wiz is no longer in the options. What's going on?

**Comment by isaacrlevin** [comment]
Wiz is broken as the dependency I have does not work after me testing extensively. Sadly I have no timetable as to when it will be fixed.
